### DISCUSSION TITLE
uap

### DISCUSSION THREAD

**Discussion by Lumiamobile** [cotaku/Bibi-Release#119] [General]
作者是否考虑修复uap的bilibili

**Comment by cotaku** [comment]
uap？是啥

**Reply by Lumiamobile** [reply]
win8.1和wp8.1的哔哩哔哩，应该也是作者大大开发的吧

**Reply by cotaku** [reply]
win8早就凉了，还有什么维护的意义……

**Comment by Lumiamobile** [comment]
可是还有wp8

在 2024-06-09 05:14:28，cotaku ***@***.***> 写道：


win8早就凉了，还有什么维护的意义……

**Reply by cotaku** [reply]
wp8只能用Silverlight开发，还是别折腾这老古董了……

**Reply by Lumiamobile** [reply]
应该只是主页和播放有问题，其他应该没事
